### PR TITLE
HL-304

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
[Link to ticket](url)

- Download abi files from github

### CODE REVIEW

**Review by bazmatic — APPROVED** [review]

**Review by Full-Node-Capital — APPROVED** [review]
LGTM

File: scripts/settle.py [hunk 1]
```diff
@@ -25,23 +25,23 @@ def get_oracle() -> str:
...
 def load_oracle():
 
     address = get_oracle()
 
-    with open('./artifacts/contracts/MarketOracle.sol/MarketOracle.json') as f:
-        data = json.load(f)
-        abi = data['abi']
-        contract = web3.eth.contract(address=address, abi=abi)
-        return contract
+    response = requests.get('https://raw.githubusercontent.com/horse-link/contracts.horse.link/main/deployments/goerli/MarketOracle.json')
```
Comment (brettvniekerk) [inline scripts/settle.py:40]: not sure what the pattern here is exactly, but the `load_market` function has an `f` followed by the url (see line 29), just wondering if that should be used here?

**Review by brettvniekerk — APPROVED** [review]
LGTM

**Comment by brettvniekerk** [comment]
@bitcoinbrisbane Full-Node-Capital / Secuaa Coin is @OptimusOpus alt account

**Pull request merged**